Gemfile

```diff
@@ -2,4 +2,5 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'activerecord', '>= 7.1.0'
+gem 'activerecord', '~> 7.2.2'
```

Comment:
If we're still compatible with 7.1, do we need this change? This will prevent us from using it with 7.1.

Reply:
I don't think so, I'll unroll this